src/rust/plugin-registry/tests/test_list_plugins.rs

```diff
@@ -0,0 +1,114 @@
+#![cfg(feature = "integration_tests")]
+
+use bytes::Bytes;
+use clap::Parser;
+use grapl_utils::future_ext::GraplFutureExt;
+use rust_proto::{
+    client_factory::{
+        build_grpc_client,
+        services::PluginRegistryClientConfig,
+    },
+    graplinc::grapl::api::plugin_registry::v1beta1::{
+        ListPluginsRequest,
+        PluginMetadata,
+        PluginType,
+    },
+};
+
+fn get_example_generator() -> Result<Bytes, std::io::Error> {
+    std::fs::read("/test-fixtures/example-generator").map(Bytes::from)
+}
+
+#[test_log::test(tokio::test)]
+async fn test_list_plugins() -> eyre::Result<()> {
+    let client_config = PluginRegistryClientConfig::parse();
+    let mut client = build_grpc_client(client_config).await?;
+
+    let tenant_id = uuid::Uuid::new_v4();
+    let event_source_1_id = uuid::Uuid::new_v4();
+    let event_source_2_id = uuid::Uuid::new_v4();
+
+    let generator_1_create_response = {
+        let display_name = "generator 1";
+        let artifact = get_example_generator()?;
```

Comment:
tbh we can just use a regular `let artifact = Bytes::from("arbitrary binary");` 

instead of the concrete, real Example Generator here, since we're not really testing the contents of the generator binary, just how they're managed.

Reply:
(it is fine as-is, but I like to use intentionally-marked arbitrary data whenever possible in a test so I know what's important and what's not)

Reply:
done